tests/integration/mon_test.go

```diff
@@ -29,21 +29,22 @@ import (
 func (suite *SmokeSuite) TestMonFailover() {
 	logger.Infof("Mon Failover Smoke Test")
 
+	namespace := defaultRookNamespace
 	opts := metav1.ListOptions{LabelSelector: "app=rook-ceph-mon"}
-	monReplicas, err := suite.k8sh.Clientset.ReplicaSets(defaultRookNamespace).List(opts)
+	monReplicas, err := suite.k8sh.Clientset.ReplicaSets(namespace).List(opts)
```

Comment:
Why not directly use `defaultRookNamespace` (also for the below usages)? This could save one line ;)

Reply:
sure.